common/src/main/java/com/wynntils/models/containers/BankModel.java

```diff
@@ -53,11 +58,17 @@ public class BankModel extends Model {
     private final Storage<Map<String, Map<Integer, String>>> customCharacterBankPagesNames =
             new Storage<>(new TreeMap<>());
 
+    public static final int LAST_BANK_PAGE_SLOT = 8;
+
+    // Test in BankModel_PERSONAL_STORAGE_PATTERN
+    private static final Pattern PERSONAL_STORAGE_PATTERN =
+            Pattern.compile("^§0\\[Pg\\. (\\d+)\\] §8[a-zA-Z0-9_ ]+'s?§0 (.*)$");
     private static final int MAX_CHARACTER_BANK_PAGES = 10;
+    private static final StyledText LAST_BANK_PAGE_STRING = StyledText.fromString(">§4>§c>§4>§c>");
 
     private boolean editingName;
     private int currentPage = 1;
-    private SearchableContainerType currentContainer;
+    private InteractiveContainerType currentContainer;
 
     public BankModel() {
```

Comment:
Maybe this should be renamed `StorageModel` now that it supports so much more than the original bank? (And the `type.isBank()` etc as well.)

Reply:
We might confuse it with our own Storage types, but since it's a model, we could tell the difference. If you do the rename, just make sure to add a comment to clarify that this is not `StorageManager`.

Reply:
Might be a bad idea. Keep it as "Bank" for now.